test/unit/app/helpers/admin/republishing_helper_test.rb

```diff
@@ -1,6 +1,38 @@
 require "test_helper"
 
 class Admin::RepublishingHelperTest < ActionView::TestCase
+  test "#republishable_content_types returns a sorted list combining valid document types and other publishable content types" do
+    # we need to eager load here to ensure we have all the models
+    Rails.application.eager_load!
+
+    stubbed_document_types = %w[DocumentTypeOne DocumentTypeTwo]
+    Document.expects(:document_types).once.returns(stubbed_document_types)
+
+    expected_non_document_republishable_content_types = %w[Contact
```

Comment:
Maybe this could just be an array that combines `DocumentTypeOne` and `DocumentTypeTwo`, to avoid the need to merge, flatten and sort on lines 28-31?  But not fussed either way if you prefer leaving as is.